inc/checks/class-validate-mime.php

```diff
@@ -0,0 +1,1279 @@
+		'uvvx'         => array( 'application/vnd.dece.unspecified' ),
+		'uvvz'         => array( 'application/vnd.dece.zip' ),
+		'uvx'          => array( 'application/vnd.dece.unspecified' ),
+		'uvz'          => array( 'application/vnd.dece.zip' ),
+		'vcard'        => array( 'text/vcard' ),
+		'vcd'          => array( 'application/x-cdlink' ),
+		'vcf'          => array( 'text/x-vcard' ),
+		'vcg'          => array( 'application/vnd.groove-vcard' ),
+		'vcs'          => array( 'text/x-vcalendar' ),
+		'vcx'          => array( 'application/vnd.vcx' ),
+		'vda'          => array( 'application/vda' ),
+		'vdo'          => array( 'video/vdo' ),
+		'vew'          => array( 'application/groupwise' ),
+		'vis'          => array( 'application/vnd.visionary' ),
+		'viv'          => array( 'video/vivo', 'video/vnd.vivo' ),
+		'vivo'         => array( 'video/vivo', 'video/vnd.vivo' ),
+		'vmd'          => array( 'application/vocaltec-media-desc' ),
+		'vmf'          => array( 'application/vocaltec-media-file' ),
+		'vob'          => array( 'video/x-ms-vob' ),
+		'voc'          => array( 'audio/voc', 'audio/x-voc' ),
+		'vor'          => array( 'application/vnd.stardivision.writer' ),
+		'vos'          => array( 'video/vosaic' ),
+		'vox'          => array( 'application/x-authorware-bin' ),
+		'vqe'          => array( 'audio/x-twinvq-plugin' ),
+		'vqf'          => array( 'audio/x-twinvq' ),
+		'vql'          => array( 'audio/x-twinvq-plugin' ),
+		'vrml'         => array( 'application/x-vrml', 'model/vrml', 'x-world/x-vrml' ),
+		'vrt'          => array( 'x-world/x-vrt' ),
+		'vsd'          => array( 'application/vnd.visio' ),
+		'vsf'          => array( 'application/vnd.vsf' ),
+		'vss'          => array( 'application/vnd.visio' ),
+		'vst'          => array( 'application/vnd.visio' ),
+		'vsw'          => array( 'application/vnd.visio' ),
+		'vtt'          => array( 'text/vtt' ),
+		'vtu'          => array( 'model/vnd.vtu' ),
+		'vxml'         => array( 'application/voicexml+xml' ),
+		'w3d'          => array( 'application/x-director' ),
+		'w60'          => array( 'application/wordperfect6.0' ),
+		'w61'          => array( 'application/wordperfect6.1' ),
+		'w6w'          => array( 'application/msword' ),
+		'wad'          => array( 'application/x-doom' ),
+		'wav'          => array( 'audio/wav', 'audio/x-wav' ),
+		'wax'          => array( 'audio/x-ms-wax' ),
+		'wb1'          => array( 'application/x-qpro' ),
+		'wbmp'         => array( 'image/vnd.wap.wbmp' ),
+		'wbs'          => array( 'application/vnd.criticaltools.wbs+xml' ),
+		'wbxml'        => array( 'application/vnd.wap.wbxml' ),
+		'wcm'          => array( 'application/vnd.ms-works' ),
+		'wdb'          => array( 'application/vnd.ms-works' ),
+		'wdp'          => array( 'image/vnd.ms-photo' ),
+		'web'          => array( 'application/vnd.xara' ),
+		'weba'         => array( 'audio/webm' ),
+		'webapp'       => array( 'application/x-web-app-manifest+j
```

Comment:
I'm not quite sure this is the same check as #88 describes...

SHouldn't this check that `$file_extension` matches `$file_mime_type`?

Reply:
I'm a bit confused :sweat_smile: ? 
For example, a file can have an extension `.jpeg` with the following mime-types : `image/jpeg`, `image/pjpeg`, in case, the extension is modified to `.png`, the mime-type will still remain the same and return an error